<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold SemiConden</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(85, 85, 0)</string>
    </property>
    <property name="text">
     <string>WELCOME TO THE SAFTEY DETECTION</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(85, 85, 0);
border-radius : 20px;
}

QPushButton:hover{

background-color :rgb(255, 85, 0);
border : 2px solid  rgb(255, 85, 0);

}
QPushButton:pressed{
background-color: rgb(85, 255, 255);
}
;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/HAMZA ABBAS/Downloads/img1 (1).png</normaloff>C:/Users/HAMZA ABBAS/Downloads/img1 (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>150</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(85, 85, 0);
border-radius : 20px;
}

QPushButton:hover{

background-color :rgb(255, 85, 0);
border : 2px solid  rgb(255, 85, 0);
}
QPushButton:pressed{
background-color: rgb(85, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/HAMZA ABBAS/Downloads/img2 golves (1).png</normaloff>C:/Users/HAMZA ABBAS/Downloads/img2 golves (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>160</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(85, 85, 0);
border-radius : 20px;
}

QPushButton:hover{

background-color :rgb(255, 85, 0);
border : 2px solid  rgb(255, 85, 0);

}
QPushButton:pressed{
background-color: rgb(85, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/HAMZA ABBAS/Downloads/img3 jacket (1).png</normaloff>C:/Users/HAMZA ABBAS/Downloads/img3 jacket (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>150</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(85, 85, 0);
border-radius : 20px;
}

QPushButton:hover{

background-color :rgb(255, 85, 0);
border : 2px solid  rgb(255, 85, 0);

}
QPushButton:pressed{
background-color: rgb(85, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/HAMZA ABBAS/Downloads/gooleslens.png</normaloff>C:/Users/HAMZA ABBAS/Downloads/gooleslens.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>150</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-80</x>
      <y>-30</y>
      <width>641</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/newPrefix/C:/Users/HAMZA ABBAS/Downloads/PPE (2).jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/HAMZA ABBAS/Downloads/PPE (2).jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(85, 85, 0);
border-radius : 20px;
}

QPushButton:hover{

background-color :rgb(255, 85, 0);
border : 2px solid  rgb(255, 85, 0);

}
QPushButton:pressed{
background-color: rgb(85, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/HAMZA ABBAS/Downloads/mask (1).png</normaloff>C:/Users/HAMZA ABBAS/Downloads/mask (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>190</width>
      <height>190</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(85, 85, 0);
border-radius : 20px;
}

QPushButton:hover{

background-color :rgb(255, 85, 0);
border : 2px solid  rgb(255, 85, 0);

}
QPushButton:pressed{
background-color: rgb(85, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/HAMZA ABBAS/Downloads/welding (1).png</normaloff>C:/Users/HAMZA ABBAS/Downloads/welding (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>460</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(85, 85, 0);
border-radius : 20px;
}

QPushButton:hover{

background-color :rgb(255, 85, 0);
border : 2px solid  rgb(255, 85, 0);

}
QPushButton:pressed{
background-color: rgb(85, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/HAMZA ABBAS/Downloads/full suit.png</normaloff>C:/Users/HAMZA ABBAS/Downloads/full suit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>190</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);

border-radius : 20px;
color:white
</string>
    </property>
    <property name="text">
     <string>Multiple selection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(255, 0, 0);
border-radius : 20px;
color:white
}

QPushButton:hover{

background-color :rgb(255, 85, 0);
border : 2px solid  rgb(255, 85, 0);

}
QPushButton:pressed{
background-color: rgb(85, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>CLEAR</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton_9</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="demo.qrc"/>
 </resources>
 <connections/>
</ui>
